src/model/transaction/AggregateTransactionCosignature.ts

```diff
@@ -15,7 +15,7 @@
  */
 
 import {PublicAccount} from '../account/PublicAccount';
-
+import { NetworkType } from '../blockchain/NetworkType';
```

Comment:
why include NetworkType in this file ?

Reply:
It's a good question, some how, it's not directly referenced in the class, but when trying to call the method, it keeps complaining missing this ref.  I will ave a look anyway, could be VS code's issue